client/my-sites/themes/themes-search-card/index.jsx

```diff
@@ -154,4 +160,20 @@ const ThemesSearchCard = React.createClass( {
 	}
 } );
 
-export default localize( ThemesSearchCard );
+export default connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const isJetpack = isJetpackSite( state, siteId );
+
+		let externalUrl;
+		if ( ! siteId ) {
+			externalUrl = oldShowcaseUrl;
+		} else if ( isJetpack ) {
+			externalUrl = getSiteAdminUrl( state, siteId, 'theme-install.php' );
```

Comment:
Good catch! I'm starting to think I've been doing this PR a little too mechanically 😄